dt-posts/dt-posts.php

```diff
@@ -1594,12 +1594,26 @@ public static function get_default_list_column_order( $post_type ){
         return $columns;
     }
 
+    private static function adjust_post_tile_labels( $tiles ): array {
+        $adjusted_titles = [];
+        foreach ( $tiles as $tile_key => $tile ) {
 
+            // Support custom label translations; or simply default to initial label entry
+            $label = ( ! empty( $tile['translations'] ) && isset( $tile['translations'][ determine_locale() ] ) ) ? $tile['translations'][ determine_locale() ] : $tile['label'];
+
+            // Update tile's label accordingly
+            $tile['label'] = $label;
+
+            $adjusted_titles[ $tile_key ] = $tile;
+        }
+
+        return $adjusted_titles;
+    }
 
     public static function get_post_tiles( $post_type, $return_cache = true ){
         $cached = wp_cache_get( $post_type . "_tile_options" );
-        if ( $return_cache && $cached ){
-            return $cached;
+        if ( $return_cache && $cached ) {
+            return self::adjust_post_tile_labels( $cached );
```

Comment:
@kodinkat If we do the translation before the cache is set, then we don't need to call the function in 2 different places.

Reply:
Ok, will take a look further upstream so as to translate prior to cache update....

Reply:
@kodinkat just in this function.